debian8/guide/system/permissions/files/permissions_important_account_files/file_permissions_etc_gshadow.rule

```diff
@@ -3,9 +3,9 @@ documentation_complete: true
 title: 'Verify Permissions and ownership on <tt>gshadow</tt> File'
 
 description: |-
-    <fileperms-desc-macro file="/etc/gshadow" perms="0640" />
-      <fileowner-desc-macro file="/etc/gshadow" owner="root" />
-      <filegroupowner-desc-macro file="/etc/gshadow" group="shadow" />
+    {{{ describe_file_permissions(file="/etc/gshadow", perms="0640") }}}
+      {{{ describe_file_owner(file="/etc/gshadow", owner="root") }}}
+      {{{ describe_file_group_owner(file="/etc/gshadow", group="shadow") }}}
```

Comment:
I love the naming of these functions, it says "describe_*". Could we please follow this pattern with the other macros that are to be used in descriptions? I think it makes it very self documenting and self assuring that you are using the right macro.